### PR TITLE
[EDU-2721] Adjust WWD journey content

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Improved wording and structure in domain journey
- Fixed a translation inconsistency in build documentation

### Additional links

[EDU-2721]: https://aziontech.atlassian.net/browse/EDU-2721?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/work-with-domains/configure-domains.mdx [hunk 1]
```diff
@@ -1,48 +1,46 @@
...
-6. Disable the **CNAME access only switch**. This will enable users to access your application using the `xxxxxxxxxx.map.azionedge.net` address.
-7. Click the **Save** button.
+1. [Access RTM](https://manager.azion.com).
+2. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines, and then select **Domain**.
+3. Click the **Add Domain** button.
+4. Give your domain a name.
+5. In **Edge Certificate**, select the **Azion SAN** certificate.
+6. In **Edge Application**, select your edge application.
+7. In the **CNAME** field, you may list [custom domain](#link-a-custom-domain-to-your-application) addresses to link to your application.
+8. Disable the **CNAME access only switch**. This will enable users to access your application using the `xxxxxxxxxx.map.azionedge.net` address.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/work-with-domains/configure-domains.mdx:28]:
```suggestion
8. Disable the **CNAME access only switch**. This'll enable users to access your application using the `xxxxxxxxxx.map.azionedge.net` address.
```

File: src/content/docs/en/pages/main-menu/work-with-domains/configure-domains.mdx [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
+9. Click the **Save** button.
 
-Wait some minutes for your new domain to propagate to the edge nodes and access your `xxxxxxxxxx.map.azionedge.net` address. You should now see your edge application online.
+Wait some minutes for your new domain to propagate to the edge nodes and access your `xxxxxxxxxx.map.azionedge.net` address to see your edge application online.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/work-with-domains/configure-domains.mdx:31]:
```suggestion
Wait a few minutes for your new domain to propagate to the edge nodes and access your `xxxxxxxxxx.map.azionedge.net` address to see your edge application online.
```

File: src/content/docs/en/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 3]
```diff
@@ -84,12 +85,14 @@ curl --location 'https://api.azionapi.net/intelligent_dns' \
 }
 ```
 
-3. Copy the values inside the `nameservers` field. These values will be used to [redirect the authoritative DNS of your domain](#redirect-authoritative-dns-servers).
-4. Also copy the `id` value for when you [associate an application to your zone](#create-edge-application-record-in-dns-zone).
+3. Copy the values inside the `nameservers` field. These values will be used to [redirect the authoritative nameservers of your domain](#step-2-redirect-authoritative-nameservers).
+4. Also copy the `id` value for when you [associate an application to your zone](#step-3-create-edge-application-record-in-dns-zone).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/work-with-domains/migrate-ns.mdx:89]:
```suggestion
4. Copy the `id` value for when you [associate an application to your zone](#step-3-create-edge-application-record-in-dns-zone).
```

File: src/content/docs/en/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 4]
```diff
@@ -118,14 +125,14 @@ To customize the domain of your edge application under your new zone, you need t
...
-5. Under **Type**, select `CNAME - Canonical name`.
-6. Under **Value**, enter the Azion domain address in FQDN format. Example: `xxxxxxxxxx.map.azionedge.net`
-7. Set the **TTL** and **Policy** as desired.
-8. Click the **Save** button.
+5. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines, and then select **Intelligent DNS**.
+6. Select the DNS zone.
+7. Navigate to the **Records** tab.
+8. Under **Name**, create a hostname or subdomain. For example: `*`.
+9. Under **Type**, select `CNAME - Canonical name`.
+10. Under **Value**, enter the Azion domain address in FQDN format. Example: `xxxxxxxxxx.map.azionedge.net`
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/work-with-domains/migrate-ns.mdx:133]:
```suggestion
10. Under **Value**, enter the Azion domain address in FQDN format. Example: `xxxxxxxxxx.map.azionedge.net`.
```

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/configurar-dominio.mdx [hunk 5]
```diff
@@ -50,8 +51,8 @@ curl --location 'https://api.azionapi.net/domains' \
     | --- | --- |
     | `name` | Define a string no valor como um nome da entrada do domínio com a Azion. Como este valor não representa um endereço de domínio, você pode modificá-lo para se adequar às suas necessidades. |
     | `cname_access_only` | Quando definido como `false`, permite o acesso via endereço Azion. |
-    | `digital_certificate_id` | Quando `null`, seleciona o certificado Azion SAN. Se você quiser associar seu próprio certificado, veja [Criar um certificado digital](/pt-br/documentacao/produtos/guias/criar-certificado-digital/). |
-    | `edge_application_id` | O ID da edge application vinculada ao domínio. Para obter o ID, veja [Definir configurações principais](/pt-br/documentacao/produtos/guias/build/definir-configuracoes-principais/). |
+    | `digital_certificate_id` | Quando `null`, seleciona o certificado Azion SAN. Você também pode [associar um certificado digital](/pt-br/documentacao/produtos/guias/criar-certificado-digital/) à seu domínio. |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/configurar-dominio.mdx:54]:
```suggestion
    | `digital_certificate_id` | Quando `null`, seleciona o certificado Azion SAN. Você também pode [associar um certificado digital](/pt-br/documentacao/produtos/guias/criar-certificado-digital/) a seu domínio. |
```

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 6]
```diff
@@ -9,14 +9,16 @@ permalink: /documentacao/produtos/guias/criar-certificado-digital/
...
 :::
 
+---
+
 ## Via RTM
 
 1. [Acesse o RTM](https://manager.azion.com).
 2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Digital Certificates**.
-3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate**, em seguida selecione a opção **Edge Certificate**. Se o seu certificado for do tipo CA, selecione **Trusted CA**.
+3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate**, em seguida selecione a opção **Edge Certificate**. Se o seu certificado for do tipo CA cert, selecione **Trusted CA**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:21]:
```suggestion
3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate**, em seguida selecione a opção **Edge Certificate**. Se o seu certificado for do tipo CA, selecione **Trusted CA**.
```

File: src/content/docs/en/pages/main-menu/work-with-domains/point-domain.mdx [hunk 7]
```diff
@@ -1,19 +1,20 @@
 ---
-title: Point domain to Azion
+title: How to point domain to Azion
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/work-with-domains/point-domain.mdx:2]:
```suggestion
title: How to point a domain to Azion
```

**Review by MarianaAguilera — APPROVED** [review]
Approved with @hannah-sk's suggestions.

**Pull request merged**